assets/css/variables.scss

```
@@ -14,17 +14,18 @@ $base-lineheight: 1.5em;
 
 $max-width: 1250px;
 
-
-$transition: .25s;
+$transition: 0.25s;
```

Stylelint (17.5.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)